README.md

```diff
@@ -133,36 +132,12 @@ To execute all tests, run:
 in the current directory.
 
 For the more fine-grained running of tests or doctests, use `bin/test`
-or respectively `bin/doctest`. The master branch is automatically tested
+or respectively `bin/doctest`. The `master` branch is automatically tested
 by GitHub Actions.
 
 To test pull requests, use
 [sympy-bot](https://github.com/sympy/sympy-bot).
```

Comment:
I'm not sure how relevant this is today. AFAIK, the SymPy bot isn't used to test PRs. Rather, its job is to inform about files that may have accidentally been added or deleted in a PR. I think we can remove this line.